modules/media/ajax/FileUpload.php

```diff
@@ -16,14 +16,29 @@
 if (isset($_GET['action'])) {
     $action = $_GET['action'];
     if ($action == "getData") {
-        echo json_encode(getUploadFields());
+        viewData();
     } else if ($action == "upload") {
         uploadFile();
     } else if ($action == "edit") {
         editFile();
     }
 }
 
+/**
+ * Handles the media view data process
+ *
+ * @return void
+ */
+function viewData()
```

Comment:
Could you move this function declaration to the bottom of the files with the other ones? It feels strange to have the functions in different places

Reply:
fixed